### PR TITLE
Material ID's, MetaItem Restructure and texture Changes

### PR DESCRIPTION
**Pull request by jude123412**
Metaitems, Textures, Dependency changes and MetaItems class restructure.

### CODE REVIEW

File: src/main/java/gtb/common/item/GTBMetaItems.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
 package gtb.common.item;
 
 import gregtech.api.items.metaitem.MetaItem;
+import gregtech.api.items.metaitem.StandardMetaItem;
 
-public class GTBMetaItems {
-
-    public static MetaItem<?>.MetaValueItem ASTRO_MINER;
```
Comment (tekcay) [inline src/main/java/gtb/common/item/GTBMetaItems.java:7]: I don't see the point in removing those constants as they make getting items much easier.

File: src/main/java/gtb/loaders/recipe/OilLine.java [hunk 2]
```diff
@@ -159,7 +159,7 @@ public static void init() {
 
         GTBRecipeMaps.FISCHER_TROPSCH_REACTOR.recipeBuilder().fluidInputs(DistilledWater.getFluid(2000))
                 .fluidInputs(RefineryGas.getFluid(8000)).notConsumable(dust, CobaltOxide, 1)
-                .output(GTBMetaItems.PARAFFIN_WAX, 1).fluidOutputs(Diesel.getFluid(1000))
+                .output(GTBMetaItems.getByNameOrId("paraffin_wax"), 1).fluidOutputs(Diesel.getFluid(1000))
```
Comment (tekcay) [inline src/main/java/gtb/loaders/recipe/OilLine.java:162]: See here you have to call it by a string which is very prone to error. The initial method won't get an error and it is much more convenient to use, just type `GTBItems`, dot and you can access all the items.

File: dependencies.gradle
Comment (tekcay) [inline dependencies.gradle:1]: this should be in another branch as it has nothing to do with the pr name, create another branch. Plus this crash has not been reported so far, can you open an issue ?

**Review by tekcay — CHANGES_REQUESTED** [review]

File: src/main/java/gtb/common/item/GTBMetaItems.java (hunk 1 above)
Comment (jude123412) [reply src/main/java/gtb/common/item/GTBMetaItems.java]: well I learned the hard way that changing that makes it harder to retrieve items

**Comment by jude123412** [comment]
Sorry Didn't mean to close the pull request, I have reverted the changes from the metaItems and will use that from now on :)

File: dependencies.gradle
Comment (jude123412) [reply dependencies.gradle:1]: Reverted the MetaItem Changes to previous version

**Comment by tekcay** [comment]
@jude123412 yeah figured it out haha

**Comment by jude123412** [comment]
yup metaitems will be made like that from now on :)

**Comment by jude123412** [comment]
should all be fixed... hopfully

File: dependencies.gradle
Comment (jude123412) [reply dependencies.gradle:1]: Interesting does the dependency changes cause a crash for you?

**Comment by jude123412** [comment]
Closing pull request as its a mess, Ill make a new one.

**Pull request closed without merging**